website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,179 @@
+| **Configuration**   | Configured within dbt Cloud environment and Job scheduler settings. | Defined in YAML format within dbt project files.   |
+
+## Define exports
+
+Exports are an additional configuration added to a saved query. They define how to materialize a saved query, along with the schema and table name.
+
+You can define exports in a YAML format as a key within the `saved_queries` configuration and in the same file as your metric definitions.  Here's an example of a saved query with an Export:
+
+<File name='semantic_model.yml'>
+
+```yaml
+saved_queries:
+  - name: your_query
+    description: your_description
+    query_params:
+      metrics:
+        - your_metric_name
+      group_bys:
+        - TimeDimension()
+        - ... # Additional group_bys
+      where:
+        - ... # Additional where clauses
+    export:
+      - name: your_export
+        config:
+          export_as: table # Options available: table, view
+          schema: your_schema # Optional - defaults to deployment schema
+          alias: some_table_name # Optional - defaults to Export name
+```
+</File>
+
+You can use the following parameters to define an Export:
+
+| Parameters | Type    | Required | Description    |
+| ------- | --------- | ---------- | ---------------- |
+| `name` | String    | Required     | Name of the Export object.    |
+| `saved-query` | String    | Required     | A name of a saved query that could be used.    |
+| `select` | List or String   | Optional    | Names of exports to be selected from the saved query. |
+| `exclude` | String  | Optional    | The names of exports to be excluded from the from the saved query. |
+| `export_as` | String  | Optional    | Type of Export to create from the `export_as` types available in the config. Options available are `table` or `view`. |
+| `schema` | String  | Optional    | Schema to use for creating the table or view. |
+| `alias` | String  | Optional    | Table alias to use to write the table or view. |
+
+## Run exports
+
+Once you've defined Export in your dbt project, the next step is to run them. There are two ways to run an Export:
+  
+1. [Run exports in development](#exports-in-development) using the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation).
+2. [Run exports in production](#exports-in-production) using the [dbt Cloud Job scheduler](/docs/deploy/job-scheduler).
+
+### Exports in Development
+
+You can run an Export in your development environment using your development credentials if you want to test the output of the Export before production. You can use the following command in the dbt Cloud CLI:
+
+```bash
+dbt sl export
+```
+
+You can also use the following command to run any Export defined for the saved query and materialize the table or view in your development environment:
+
+```bash
+dbt sl export --saved-query sq_name
+```
+
+---- ADD OUTPUT OF THE COMMAND HERE ----
+
+### Use the `select` flag
```

Comment:
sugg: per [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#text-formatting) tho we do have a lot of prior art with multiple decorations

```suggestion
### Use the select flag
```